test/MemorySecretTest.cs

```diff
@@ -108,18 +123,10 @@ public void TestEquals()
 		[Test]
 		public void TestNotEquals()
 		{
-			var s2 = new MemorySecret()
-			{
-				Region = GameRegion.US,
-				TargetGame = Game.Ages,
-				GameID = 14129,
-				Memory = Memory.GraveyardFairy,
-				IsReturnSecret = true
-			};
-
-			Assert.AreNotEqual(desiredSecret, s2);
-			Assert.AreNotEqual(desiredSecret, null);
-			Assert.AreNotEqual(desiredSecret, "");
+			Assert.AreNotEqual(desiredSecret, new MemorySecret());
+			Assert.False(new MemorySecret().Equals(new TestSecret()));
+			Assert.False(new MemorySecret().Equals(new GameSecret()));
+			Assert.False(new MemorySecret().Equals(null));
```

Comment:
You have a mix of code styles here. For consistency with existing code, use the `Assert.AreNotEqual` method instead, unless you want to rewrite everything to use the newer `Assert.That` style documented here: https://github.com/nunit/docs/wiki/Assertions